app/main.py

```diff
@@ -1,8 +1,67 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str) -> None:
```

Comment:
Place `) -> None:` on a new line as it is done for `__init__` in the `CarWashStation`.